src/gui/base/InfoBanner.ts

```diff
@@ -41,10 +41,11 @@ export class InfoBanner implements Component<InfoBannerAttrs> {
 				},
 			},
 			[
-				m(".flex", [
-					m(".mt-s.mr-s", this.renderIcon(icon, type ?? null)),
-					m(".flex-grow", [
-						m(".mr.pt-s", [m(".small", lang.getMaybeLazy(message))]),
+				m(".mt-s.mr-s.abs", this.renderIcon(icon, type ?? null)), // absolute position makes the icon fixed to the top left corner of the banner
+				m("",
+					{style: {margin: "0 0 0 25px"}}, // allow room for the icon
```

Comment:
if the size of icons changes, this will break. we can use the `size` object to relate this margin to the icon's size with `margin-left: px(size.icon_size_large + 1)`

Reply:
Fixed!